src/pages/appointmentsTest/components/OnClickDateModal.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Modal } from "antd";
+import dayjs from "dayjs";
+import Strings from "../../../utils/Strings";
+
+interface OnClickDateModalProps {
+  isVisible: boolean;
+  onCancel: () => void;
+  clickedDate: Date | null;
+}
+
+const OnClickDateModal: React.FC<OnClickDateModalProps> = ({
+  isVisible,
+  onCancel,
+  clickedDate,
+}) => {
+  return (
+    <Modal title={Strings.selectedDate} open={isVisible} onCancel={onCancel}>
```

Comment:
I saw that this is the modal, two things, first one don't forget to add the 'Test" as i mentioned in the ticket for the modals, second one since this is a modal you can name it like RegisterAppointmentModalTest but you can do it in the next ticket just don't forget it